packages/frontier-plugins/plugin-qa/README.md

```diff
@@ -1,3 +1,12 @@
+<div align="left">
+  <br/>
+  <a href="https://www.realdecoy.com/jamaica/" title="REALDECOY">
+    <img width=400px src="https://www.realdecoy.com/wp-content/uploads/2019/02/Realdecoy-logo-transparent.png" alt="rd logo">
+  </a>
+  <br/>
+</div>
+
+
 # rdspec
```

Comment:
# Spectre

### @rdfrontier/plugin-typekit

(were deprecating references to rdspec)